src/test-utils/constants.tsx

```diff
@@ -0,0 +1,29 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { FeeAmount, Pool } from '@uniswap/v3-sdk'
+import JSBI from 'jsbi'
+
+export const testToken1 = new Token(1, '0x0000000000000000000000000000000000000001', 18)
+export const testToken2 = new Token(1, '0x0000000000000000000000000000000000000002', 18)
+export const testToken3 = new Token(1, '0x0000000000000000000000000000000000000003', 18)
```

Comment:
for these to be fully-reusable in other future tests, would it help for these to have names and symbols?

Reply:
i.e. snapshots of components that render token symbols / names

Reply:
good call, I add symbols in later PRs (the actual test prs) but makes sense to add them all here, done.